packages/siwe/exports/index.ts

```diff
@@ -0,0 +1,9 @@
+import type { SIWEConfig } from '@web3modal/core'
+import { Web3ModalSIWEClient } from '../src/client.js'
+
+export { Web3ModalSIWEClient }
```

Comment:
you can remove imports and just do `export {SIWEConfig} from "@web3modal/core"`. Same for others. But looking further down the file, I think only thing you want to export from this file is `createSIWEConfig`?

And if you still need to export `Web3ModalSIWEClient` and `SIWEConfig` you can do so directly from package index.

Reply:
The use case I'm thinking about here for exporting `type SIWEConfig` is for developers to be able to define a siweConfig outside of `createSIWEConfig` and still get type safety.
See example below:

```ts
// SIWEConfig can be auto-imported because it's exported by @web3modal/siwe
const mySiweConfig: SIWEConfig = {
  createMessage: () => console.log('create SIWE message')
  // ... get type safety and auto complete on other options
}

createSIWEConfig(mySiweConfig)
```